_headers

```diff
@@ -1,4 +1,5 @@
 /*
+  Strict-Transport-Security: max-age=63072000; includeSubDomains; preload
```

Comment:
Isn't 2 years a bit long for HSTS? I usually see it set to 6 months. Fine either way, just checking.

Reply:
That's what https://hstspreload.org/ recommends, so do the netlify docs.

Reply:
:shipit: